<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SAM4C Intel Hex to ASCII for OTA Update</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="selectHexFilePushButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>91</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Hex File</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fileNameLineEdit">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>60</y>
      <width>481</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameOfFileLineEdit">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>150</y>
      <width>481</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>160</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Text File</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>290</y>
      <width>651</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>780</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSerial_Port">
    <property name="title">
     <string>Serial Port</string>
    </property>
   </widget>
   <addaction name="menuSerial_Port"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>selectHexFilePushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>selectHexFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>selectHexFile()</slot>
  <slot>programFile()</slot>
  <slot>selectTextFile()</slot>
 </slots>
</ui>
